### PR TITLE
Add Romanian language and update translations doc.

### PR DESCRIPTION
**Pull request by dykwiat**
## Description
This is to add Romanian to video js. 

## Specific Changes proposed
Please list the specific changes involved in this pull request.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [x] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [x] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

Tested in chrome by changing the sandbox inbox.html to use romanian:

```
  // added to the script section
  <script src="../dist/lang/ro.js"></script>

  // changed player to use Romanian
  var player = videojs(vid, {language: 'ro'});
```

Screen Shot 2021-07-01 at 4 26 33 PM
Screen Shot 2021-07-01 at 4 26 27 PM
Screen Shot 2021-07-01 at 4 26 44 PM

### CODE REVIEW

File: docs/translations-needed.md [hunk 1]
```diff
@@ -329,8 +329,7 @@ This default value is hardcoded as a default to the localize method in the SeekB
 |                         | {1} is loading.                                                                     |
 |                         | Exit Picture-in-Picture                                                             |
 |                         | Picture-in-Picture                                                                  |
-| es.json (missing 2)     | Exit Picture-in-Picture                                                             |
-|                         | Picture-in-Picture                                                                  |
+| es.json (Complete)      |                                                                                     |
```
Comment (dykwiat) [inline docs/translations-needed.md:332]: This was picked up when running: `npm run docs:lang` It must have been missed with a bug fix.

**Review by gkatsev — APPROVED** [review]
Thanks!

**Review by bcbclifford — APPROVED** [review]

**Review by bcbclifford — APPROVED** [review]

**Comment by dykwiat** [comment]
@gkatsev @bcbclifford - Anything else I need to do to get this merged?

**Comment by gkatsev** [comment]
@dykwiat nothing on your end. Since it adds a new language, we generally release that as a minor, but I'd like to do a patch release first.

**Comment by dykwiat** [comment]
Sounds good @gkatsev ! Thanks!

**Review by brandonocasey — APPROVED** [review]

**Pull request merged**